docs/source/build_app/build-application.rst

```diff
@@ -14,13 +14,20 @@
 .. * limitations under the License.
 .. *******************************************************************************/
 
-Build application with oneDAL
-=============================
+Build applications with oneDAL
+==============================
 
 See Get Started Guides for `Windows\* <https://software.intel.com/en-us/get-started-with-daal-for-windows>`_ 
 and `Linux\* <https://software.intel.com/en-us/get-started-with-daal-for-linux>`_ 
 for instruction on how to build applications for C++.
-This section contains instructions for building applications with oneDAL for SYCL\*.
+This section contains instructions for building applications with |short_name| for SYCL\*.
+
+.. warning::
```

Comment:
Might be we should replace this with just "note"?  warning looks too dangerous =)

Reply:
made it less dangerously-looking